### DISCUSSION TITLE
[Feature] Find and Remove Duplicate between users' library

### DISCUSSION THREAD

**Discussion by athiosh** [immich-app/immich#10407] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

With the new duplicate find feature, it would be great if it can expand to between users. My wife and I sent a lot of photos of our pets and kids when one of us is out of town and this caused us having the same photo but just in our own libraries. It will be great if we can find those and remove from one user library.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by blublub** [comment]
I would mark it as a duplicate to my post yesterday as we are talking about the same functionality:

https://github.com/immich-app/immich/discussions/10360
